requirements/dev.in

```diff
@@ -1,2 +1,3 @@
 -r base.txt
 pylint
+coverage
```

Comment:
I recommend to keep the entries sorted for maintenability.

Reply:
Addressed https://github.com/kpi-web-guild/django-girls-blog-Martolivna/pull/21/commits/f5f6ceeeeae84a95111d275b0669193609030367 .

Reply:
Marking this as resolved.